www/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Weekly Weather</title>
+    <meta name="viewport"
+	  content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
+    
+    <link href="lib/ionic/css/ionic.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+    
+    <!-- ionic/angularjs js -->
+    <script src="lib/ionic/js/ionic.bundle.js"></script>
+    <script src="lib/ionic/js/angular/angular-resource.js"></script>
+    
+    <!-- cordova script (this will be a 404 during development) -->
+    <script src="cordova.js"></script>
+    
+    <!-- your app's js -->
+    <script src="js/app.js"></script>
+    <script src="js/services.js"></script>
+    <script src="js/controllers.js"></script>
+    
+  </head>
+  <body ng-app="weather">
+				
+		
+
+    <ion-header-bar class="bar-dark">
+      <h1 class="title">Wenham, MA</h1>
+    </ion-header-bar>
+
+    <!-- <ion-nav-bar class="bar-positive">
+    </ion-nav-bar> -->
+    
+    <ion-pane ng-controller="MainCtrl">
+	<ion-content scroll="false">
+	  <div class="row row-67 center">
+	    <div class="col">
+	      <script>console.log("today")</script>
+	      Current Temperature: {{tempCurrent}} °F<br>
+				Today's High: {{tempDayHigh}} °F<br>
+				Today's Low: {{tempDayLow}} °F<br>
+				Current Forecast: {{summary}}<br>
+				{{humidity*100}}% humidity<br>
+				Wind speed: {{wind}} mph<br>
+				Day of the week: {{currentDay}}<br>
+				{{icon}}
+	    </div>
+	  </div>
+	  <div class="row row-33">
+			<div class="col center">
+				High:<br>
+				Low:
+			</div>
+	    <div class="col center">{{currentDay1}}<br>
+	      {{tempDay1High}} °F<br>
+				{{tempDay1Low}} °F</div>
+			<div class="col center">{{currentDay2}}<br>
+	      {{tempDay2High}} °F<br>
+				{{tempDay2Low}} °F</div>
+			<div class="col center">{{currentDay3}}<br>
+	      {{tempDay3High}} °F<br>
+				{{tempDay3Low}} °F</div>
+			<div class="col center">{{currentDay4}}<br>
+	      {{tempDay4High}} °F<br>
+				{{tempDay4Low}} °F</div>
+			<div class="col center">{{currentDay5}}<br>
+	      {{tempDay5High}} °F<br>
+				{{tempDay5Low}} °F</div>
+			<div class="col center">{{currentDay6}}<br>
+	      {{tempDay6High}} °F<br>
+				{{tempDay6Low}} °F</div>
+			<div class="col center">{{currentDay7}}<br>
+	      {{tempDay7High}} °F<br>
+				{{tempDay7Low}} °F</div>
+	  </div>
+	</ion-content>
+      </ion-pane>
+
+  </body>
+</html>
```

Comment:
Index looks good. I second what Cody has said. Create some white space to make it a little easier to read.